pkg/kubelet/allocation/state/state.go

```diff
@@ -23,10 +23,7 @@ import (
 )
 
 // PodResourceAllocation type is used in tracking resources allocated to pod's containers
-type PodResourceAllocation map[string]map[string]v1.ResourceRequirements
-
-// PodResizeStatus type is used in tracking the last resize decision for pod
-type PodResizeStatus map[string]v1.PodResizeStatus
```

Comment:
Note to reviewers: this was accidentally copied over in https://github.com/kubernetes/kubernetes/pull/130254, but is unused.